### PR TITLE
DAOS-5685 il: Force use of gcc for the interception library.

### PR DESCRIPTION
**Pull request by ashleypittman**
clang is missing some symbols, so force use of gcc until we
can resolve this.

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

File: src/SConscript [hunk 1]
```diff
@@ -47,6 +47,8 @@ def scons():
     # Generic DAOS includes
     env.AppendUnique(CPPPATH=[Dir('include').srcnode()])
     env.AppendUnique(CPPPATH=[Dir('include')])
+    il_env.AppendUnique(CPPPATH=[Dir('include').srcnode()])
```
Comment (jolivier23) [inline src/SConscript:50]: I think Dir('include') will add both paths to command line

Comment (ashleypittman) [reply src/SConscript:50]: I had some trouble with this PR, once it's working I will try selectively backing parts out to check things this.

Comment (ashleypittman) [reply src/SConscript:50]: It needs both.  The srcnode to pick up the source, and the one without to pick up the daos_version.h

**Comment by ashleypittman** [comment]
Assuming this latest version passes I think I've done as much as I can with this.

**Review by daltonbohning — COMMENTED** [review]
This LGTM, but I'm not familiar enough with the inner workings of scons to know if anything is "wrong"

**Review by jolivier23 — APPROVED** [review]

**Comment by kprantis** [comment]
Review was requested on this 9 days ago, but still no second review?

**Review by johannlombardi — APPROVED** [review]

**Comment by ashleypittman** [comment]
> Review was requested on this 9 days ago, but still no second review?

I discussed this with Scott earlier, it's not easy getting people to review build patches as only Jeff and to a lesser extent myself ever touch this.

**Pull request merged**